internal/wasm/call_context.go

```diff
@@ -74,12 +74,12 @@ func (m *CallContext) String() string {
 }
 
 // Close implements the same method as documented on api.Module.
-func (m *CallContext) Close() (err error) {
-	return m.CloseWithExitCode(0)
+func (m *CallContext) Close(ctx context.Context) (err error) {
+	return m.CloseWithExitCode(ctx, 0)
 }
 
 // CloseWithExitCode implements the same method as documented on api.Module.
-func (m *CallContext) CloseWithExitCode(exitCode uint32) (err error) {
+func (m *CallContext) CloseWithExitCode(_ context.Context, exitCode uint32) (err error) {
```

Comment:
Do you think it's worth adding a comment along the lines of `// Currently context isn't used, but if using in the future, don't forget to convert nil to context.Background`?

Reply:
Though we probably don't need a comment if we add a unit test now passing in `nil` which would presumably fail if used  in the future. Not sure if I see the test but maybe just missing it.

Reply:
sure I can do that

Reply:
I both changed unit tests to pass nil and not nil, as well added the note you mentioned. Good idea!